tools/model_explorer_circle/src/model_explorer_circle/main.py

```diff
@@ -40,6 +40,8 @@ def __init__(self):
             v: k
             for k, v in circle_schema.TensorType.__dict__.items()
         }
+        # Number of elements to show in tensor values (default: 16)
+        self.config_element_size = 16
```

Comment:
The added code below is used `const_element_count_limit`, 
```
        if settings.get('const_element_count_limit'):
            self.const_element_count_limit = settings['const_element_count_limit']
```
but,  this is `config_element_size`. are they variables for different purposes?

Reply:
Oh, I missed to update it while I change the variable name.
Thanks 👍